packages/device_info_plus/device_info_plus/ios/Classes/FPPDeviceInfoPlusPlugin.m

```diff
@@ -21,8 +21,10 @@ - (void)handleMethodCall:(FlutterMethodCall *)call
     struct utsname un;
     uname(&un);
 
+    NSNumber *isPhysicalNumber =
+        [NSNumber numberWithBool:[self isDevicePhysical]];
```

Comment:
Objective-C is weird :)

Reply:
It was my first time writing code in Objective-C, from a Java and Android background. Undoubtedly, it was horrible for me.